src/main/java/Duke.java

```diff
@@ -1,10 +1,228 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final String delete_task_number_missing = "Please enter a valid task number";
+    private static final String dash = ("\u2500").repeat(5);
+    public static final String line = "//";
+    private static final String greeting = "Hello! I'm Duke \n" +
+            "What can I do for you?";
+    private static final String task_read = "Here are the tasks in your list: \n";
+    private static final String task_added = "Got it. I've added this task: \n";
+    private static final String task_completed = "Nice! I've marked this task as complete. \n";
+    private static final String farewell = "Bye. Hope to see you again soon.";
+    private static final String task_index_out_of_bounds = "That task does not exist.";
+    private static final String task_number_format = "Please enter the task number in numerals.";
+    public static final String task_no_description = "Invalid, no task description provided.";
+    public static final String task_invalid_type = "Invalid, not an accepted task type";
+    public static final String task_list_number = "\nNow you have %d tasks in the list";
+    public static final String task_removed = "Noted. I've removed this task";
```

Comment:
Perhaps you can declare them as constants and follow the coding violations. eg. static final String TASK_REMOVED = ...